### PR TITLE
Build release on push to main

### PR DESCRIPTION
**Pull request by jacobcrawford**

### CODE REVIEW

File: templates/github-workflows/build_release_helm_chart.yaml [hunk 1]
```diff
@@ -1,74 +1,18 @@
...
-      - name: Update Helm Chart version
-        if: ${{ !env.USE_QUOTES_HELM }} 
-        run: |
-          sed -i 's/version: v.*/version: ${{ steps.get-latest-tag.outputs.tag }}/g' chart/Chart.yaml
-
-      - name: Commit Helm version
-        uses: stefanzweifel/git-auto-commit-action@v4
-        with: 
-          commit_message: "New helm chart version ${{ steps.get-latest-tag.outputs.tag }}"
-          tagging_message: ${{ steps.get-latest-tag.outputs.tag }}
```
Comment (skovbakke) [inline templates/github-workflows/build_release_helm_chart.yaml:71]: By deleting this you no loger create a git tag on the release.

**Review by skovbakke — CHANGES_REQUESTED** [review]
I don't think we have a git tag anymore.

File: templates/github-workflows/build_release_helm_chart.yaml [hunk 2]
```diff
@@ -84,8 +28,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with: 
-          tag_name: ${{ steps.get-latest-tag.outputs.tag }}
-          release_name: Release ${{ steps.get-latest-tag.outputs.tag }}
+          tag_name: ${{ env.HELM_VERSION }}
+          release_name: Release ${{ env.HELM_VERSION }}
```
Comment (jacobcrawford) [inline templates/github-workflows/build_release_helm_chart.yaml:32]: The tag is created with the release as it does not exist before the release.

Comment (skovbakke) [reply templates/github-workflows/build_release_helm_chart.yaml:32]: I see. I thought tags were created separate from the release. This is good 😄

**Review by MartinSchmidt — APPROVED** [review]

**Pull request merged**